### PR TITLE
Add LREM to redis lib

### PR DESCRIPTION
**Pull request by shenoybr**
@kevin-cantwell :eyes: 
@kjsteuer 
Adding LREM wrapper to pool and connection.

### CODE REVIEW

File: redis/connection.go [hunk 1]
```diff
@@ -276,6 +276,11 @@ func (s *connection) RPush(key string, values ...string) (int, error) {
 	return redigo.Int(s.Do("RPUSH", redigo.Args{key}.AddFlat(values)...))
 }
 
+func (s *connection) LRem(key string, value string) error {
```
Comment (kevin-cantwell) [inline redis/connection.go]: Oops, look like you forgot the count param.

**Comment by shenoybr** [comment]
@kevin-cantwell fixed

**Comment by kevin-cantwell** [comment]
I don't see it?

**Comment by shenoybr** [comment]
force push

File: redis/redis.go [hunk 2]
```diff
@@ -120,6 +120,7 @@ type ListCommands interface {
 	LPush(key string, values ...string) (int, error)
 	LTrim(key string, startIndex int, endIndex int) error
 	LRange(key string, startIndex int, endIndex int) ([]string, error)
+	LRem(key string, value string) (int, error)
```
Comment (kevin-cantwell) [inline redis/redis.go]: `LRem(key string, value string, count int) (int, error)`

**Pull request merged**